configurator/config/config.go

```diff
@@ -77,7 +77,15 @@ func (c *PMMConfig) setDefaultValues() {
 func (c *PMMConfig) parseConfig() {
 	// parseConfig() runs before setDefaultValues(), so it is needed to set default manually
 	if c.ConfigPath == "" {
-		c.ConfigPath = "/srv/update/pmm-manage.yml"
+		_, configReadErr := os.Stat("/srv/update/pmm-manage.yml")
+		if configReadErr == nil {
+			c.ConfigPath = "/srv/update/pmm-manage.yml"
+		} else {
+			_, testConfigReadErr := os.Stat("./tests/sandbox/config.yml")
+			if testConfigReadErr == nil {
+				c.ConfigPath = "./tests/sandbox/config.yml"
+			}
+		}
```

Comment:
Better would be env variable, and if not set then fallback to default `/srv/update/pmm-manage.yml`. Now you have code/logic dupe.

Reply:
fixed